kcb-ci-runner.sh

```diff
@@ -0,0 +1,80 @@
+#!/bin/bash 
+set -e
+
+export REPORTS_HOME=$USER_HOME/reports
+export MINIKUBE_HOME=$PROJECT_HOME/provision/minikube
+
+echo "INFO: reset the keycloak based on the parametrized input"
+export RESET_KEYCLOAK=${RESET_KEYCLOAK:-true}
+if $RESET_KEYCLOAK; then
+  cd $MINIKUBE_HOME
+  task reset-keycloak
+  echo "INFO: task reset-keycloak has been run"
+  cd $PROJECT_HOME
+else
+  echo "INFO: Keycloak DB is not reset for this run"
+fi
+
+echo "INFO: set environment variables with parametrized inputs or defaults"
+export GRAPHITE_TCP_ADDR=$(minikube ip)
+export KC_SERVER_URL=${KC_SERVER_URL:-"https://keycloak.$GRAPHITE_TCP_ADDR.nip.io/"}
+export SCENARIO=${SCENARIO}
+export CLIENT_SECRET=setup-for-benchmark
+export RAMPUP=${RAMPUP}
+export MEASUREMENT=${MEASUREMENT}
+export RAMPDOWN=${RAMPDOWN}
+export USER_THINK_TIME=${USER_THINK_TIME}
+export SLA_ERROR_PERCENTAGE=${SLA_ERROR_PERCENTAGE:-0}
+export SLA_MEAN_RESPONSE_TIME=${SLA_MEAN_RESPONSE_TIME:-400}
+
+export WORKLOAD_MODEL=${WORKLOAD_MODEL}
+if [ "$WORKLOAD_MODEL" = "open" ]; then 
+    export WORKLOAD_PARAM="--users-per-sec=$WORKLOAD_UNIT"
+elif [ "$WORKLOAD_MODEL" = "closed" ]; then 
+    export WORKLOAD_PARAM="--concurrent-users=$WORKLOAD_UNIT"
+else 
+    echo "Invalid WORKLOAD_MODEL: \"$WORKLOAD_MODEL\". Valid values: \"open\" or \"closed\"."
+    exit 1
+fi
+
+#pull latest project and build
+echo "Building '$KCB_REPOSITORY' branch 'main'"
+git diff
+git pull
+export KCB_REVISION=$(git rev-parse --short HEAD)
+echo "KCB_REVISION: $KCB_REVISION"
+mvn -f benchmark clean install -DskipTests
```

Comment:
We're running in CI here, please add a "-B" to the Maven command line to minimize log output.

Reply:
sure thing, added it